gui/gamesetup/gamesetup~!extra_commands.js

```diff
@@ -454,697 +300,276 @@ g_NetworkCommands["/countdown"] = (input) => {
-  pMainland_1v1_defaults();
-};
-g_NetworkCommands["/p1"] = (text) => {
-  pMainland_1v1_defaults();
-};
-g_NetworkCommands["/pMainland_2v2_defaults"] = (text) => {
-  pMainland_defaults(2);
-};
-g_NetworkCommands["/p2Mainland_defaults"] = (text) => {
-  pMainland_defaults(2);
+  const key = 'autocivP.gamesetup.lastCommandProfile';
+  let lastProfile = Engine.ConfigDB_GetValue("user", key);
+  if (lastProfile === '/pRestoreLastProfile') lastProfile = '';
+  selfMessage(`Your last used profile was: ${lastProfile}`);
+  const chatInput = Engine.GetGUIObjectByName("chatInput");
+  chatInput.focus();
+  chatInput.caption = lastProfile || '/help mainland';
 };
-g_NetworkCommands["/2Mainland_defaults"] = (text) => {
-  pMainland_defaults(2);
-};
-g_NetworkCommands["/p3Mainland_defaults"] = (text) => {
-  pMainland_defaults(3);
-};
-g_NetworkCommands["/3Mainland_defaults"] = (text) => {
-  pMainland_defaults(3);
-};
-g_NetworkCommands["/p4Mainland_defaults"] = (text) => {
-  pMainland_defaults(4);
-};
-g_NetworkCommands["/4Mainland_defaults"] = (text) => {
-  pMainland_defaults(4);
-};
-g_NetworkCommands["/pMBMainland_2v2_defaults"] = (text) => {
-  pMBMainland_2v2_defaults();
-};
-g_NetworkCommands["/pVolcano_defaults"] = (text) => {
-  pVolcano_defaults();
-};
-g_NetworkCommands["/pVolcano_Extrem"] = (text) => { // a often happening typo . it should be extinct_volcano but be more tollerant
-  pVolcano_Extrem();
-};
-g_NetworkCommands["/pVolcano_ExtremExtreme"] = (text) => { // a often happening typo . it should be extinct_volcano but be more tollerant
-  pVolcano_ExtremExtreme();
-};
-g_NetworkCommands["/pUnknown_defaults"] = (text) => {
-  pUnknown();
-};
-g_NetworkCommands["/pPolarSeaTheWolfesMap"] = (text) => {
-  pPolarSeaTheWolfesMap();
-};
-g_NetworkCommands["/pWolfesInPolarSea"] = (text) => {
-  pPolarSeaTheWolfesMap();
-};
-  /*
-Jitsi for Quick Team Calls
-Jitsi is a great way to have quick team calls without any setup process. It can also be used as an audio chat for your 0ad-team.
-
-Jitsi is an easy, no-setup way to have quick team calls and audio chats. Perfect for 0ad-teams.
-
-Jitsi: Quick team calls, no setup, audio chat.
-
-*/
-
 
-// g_NetworkCommands["/jitsiBasic"] = (text) => {
-//   if (g_linkLong == null) {
-//     let linkidShort = Date.now().toString().substring(10);    // not open this link always. if you have it already probably
-//     g_linkLong = `https://meet.jit.si/0ad${linkidShort}audio`;
-//     openURL(g_linkLong);
-//   }
-//   let linkTeam1example = `${g_linkLong}team123`;
-//   selfMessage(
-//     ` recommendation: send later in your private team-game-chat a other unique link for audio chat. Example:  ${linkTeam1example}`
-//   );
-//   selfMessage(`${g_linkLong}`);
-// };
-
-g_NetworkCommands["/team"] = (text) => game.set.teams(text);
-// g_NetworkCommands["/1v1"] = () => game.set.teams("team 1v1");
-// g_NetworkCommands["/2v2"] = () => game.set.teams("team 2v2");
```

Comment:
Thanks a lot. Maybe a bit too much. For example here. I find it useful to be able to switch teams so quickly. But if you find it unnecessary and annoying, fine. I was happy to use it sometimes.